commercetools-models/src/test/java/io/sphere/sdk/products/commands/ProductUpdateCommandIntegrationTest.java

```diff
@@ -8,7 +8,10 @@
 import io.sphere.sdk.commands.UpdateAction;
```

Comment:
Please remove the `StagedWrapper` and then the `possibleToHackUpdateForStagedAndCurrent`: we don't need them anymore with your fix :wink: